app/src/main/java/de/westnordost/streetcomplete/quests/QuestModule.java

```diff
@@ -93,6 +93,7 @@
 				// ↓ 2. important data that is used by many data consumers
 				new AddRoadName(o, roadNameSuggestionsDao, putRoadNameSuggestionsHandler),
 				new AddPlaceName(o, featureDictionaryFuture),
+				new AddBusStopName(o),
```

Comment:
I find AddOneway more important than bus stop name, but otherwise it is fine

Reply:
Changed as requested. Given rarity of oneway quest it makes little difference, though I am curious what makes bus stop name less important than AddPlaceName.